hw/ip/rv_core_ibex/README.md

```diff
@@ -1,6 +1,10 @@
 # Ibex RISC-V Core Wrapper Technical Specification
 
-{{#block-dashboard rv_core_ibex}}
+[`rv_core_ibex`](https://ibex.reports.lowrisc.org/opentitan/latest/report.html):
+![](https://dashboards.lowrisc.org/badges/dv/rv_core_ibex/test.svg)
+![](https://dashboards.lowrisc.org/badges/dv/rv_core_ibex/passing.svg)
+![](https://dashboards.lowrisc.org/badges/dv/rv_core_ibex/functional.svg)
+![](https://dashboards.lowrisc.org/badges/dv/rv_core_ibex/code.svg)
```

Comment:
The ibex badges are currently broken.

Reply:
They are now available under https://dashboards.lowrisc.org/badges/dv/ibex/opentitan/test.svg ![img](https://dashboards.lowrisc.org/badges/dv/ibex/opentitan/test.svg)

Reply:
Thanks!